.github/workflows/prettier-ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: Prettier CI
+
+on:
```

Comment:
I think it's a good idea to update the trigger events to include push to `main` and manual i.e.,

```
on:
  push:
    branches:
      - main
  pull_request:
  workflow_dispatch:
  ```
  We've had issues with CIs passing on PR and breaking right after they were merged into `main` and it's nice to have the ability to trigger the workflow manually should the need ever arises.